base_export_manager/models/ir_exports_line.py

```diff
@@ -142,8 +142,8 @@ def _inverse_name(self):
             if any(parts):
                 # invalidate_cache -> in order to get actual value of field 'label'
                 # in function '_check_name'
-                self.invalidate_cache(ids=one.ids)
-                one._check_name()
+                self.env.flush_all()
+                self.env.invalidate_all()
```

Comment:
Passing `flush=True` parameter to invalidate_all() wouldn't be the same as both lines that you wrote?
https://github.com/odoo/odoo/blob/16.0/odoo/api.py#L714-L723